frontend/src/layout/navigation/CloudAnnouncement.tsx

```diff
@@ -1,10 +1,24 @@
 import React from 'react'
-import { Alert } from 'antd'
+import { Alert, Space, Button } from 'antd'
 
 export function CloudAnnouncement({ message }: { message: string }): JSX.Element | null {
+    const parsed_message = message.split('_').join(' ');
+    const github_issue_regex = /ph-([0-9]{1,8})/ig;
+    const github_issue_ids = github_issue_regex.exec(message);
```

Comment:
no need for `;`, but the pre-commit linter should have fixed that

Reply:
Done...things have changed since I last wrote JS haha.